openshift-ci/run_frrk8s_e2e.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/bash
+set -ex
+
+FRRK8S_DIR="$(dirname $(readlink -f $0))/../../frr"
+
+KUBECONFIG=$(readlink -f ../../ocp/ostest/auth/kubeconfig)
+pushd $FRRK8S_DIR
+
+SKIP="Leaked.*advertising\|receive.*ips.*from.*some\|VRF.*Advertise.*a.*subset.*of.*ips"
+SKIP="$SKIP\|should.*block.*always.*block.*cidr"
+
+if [[ "$BGP_TYPE" == "frr-k8s" ]]; then
+  SKIP="$SKIP\|metrics"  # because when running as a metallb pod the metrics are overridden.
+fi
+
+FRRK8S_NAMESPACE="metallb-system"
+if [[ "$BGP_TYPE" == "frr-k8s-cno" ]]; then
+  FRRK8S_NAMESPACE="openshift-frr-k8s"
+fi
+
+
+
+if [ "${IP_STACK}" = "v4" ]; then
+	SKIP="$SKIP\|IPV6\|DUALSTACK"
+	export PROVISIONING_HOST_EXTERNAL_IPV4=${PROVISIONING_HOST_EXTERNAL_IP}
+	export PROVISIONING_HOST_EXTERNAL_IPV6=1111:1:1::1
+elif [ "${IP_STACK}" = "v6" ]; then
+	SKIP="$SKIP\|IPV4\|DUALSTACK"
+	export PROVISIONING_HOST_EXTERNAL_IPV6=${PROVISIONING_HOST_EXTERNAL_IP}
+	export PROVISIONING_HOST_EXTERNAL_IPV4=1.1.1.1
+elif [ "${IP_STACK}" = "v4v6" ]; then
+	SKIP="$SKIP\|IPV6"
+	export PROVISIONING_HOST_EXTERNAL_IPV4=${PROVISIONING_HOST_EXTERNAL_IP}
+	export PROVISIONING_HOST_EXTERNAL_IPV6=1111:1:1::1
+fi
+echo "Skipping ${SKIP}"
+
+## In order to make the VRF leaking tests work we need to
+pods=$(oc get pods -l "app=frr-k8s" -n $FRRK8S_NAMESPACE -o jsonpath='{.items[*].metadata.name}')
+
+echo "creating vrfs in pods $pods"
```

Comment:
question: are those pods host network or not? Would the vrf can be created also in node `oc get nodes` ?

Reply:
frr-k8s is host networked, otherwise we wouldn't be able to mess up with the host.  Not sure I understood the other question about `oc get nodes`

Reply:
but oc debug will open an interactive shell on the node (or, on a pod running on that node), or is there a way to run a command with it?

Reply:
```
oc debug node/w1 --image quay.io/karampok/snife:latest 2>/dev/null -- ip --br a s br-ex
br-ex            UNKNOWN        10.10.10.104/24 169.254.0.2/17 10.10.10.126/32 2600:52:52:52::126/128 fd69::2/112 2600:52:52:52:dcad:beff:feff:1190/64 fe80::dcad:beff:feff:1190/64
```
there are also some no tty args

Reply:
or `oc debug node/w0 -- chroot /host sh -c 'iptables-save'` is you want to find  the binaries on the host